content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "DBT models for CHT Applications"
+linkTitle: "DBT Models"
+weight: 2
+description: >
+  Guide for building DBT models for CHT applications
+relatedContent: >
+  core/overview/db-schema
+  apps/reference/app-settings/hierarchy
+---
+
+## Overview
+
+[CHT Sync]({{< relref "core/overview/cht-sync" >}}) copies data from CouchDB to a relational database. It initially stores the document data from CouchDB in a `jsonb` column in a single table. This is not possible to query for analytics, so it uses [DBT](https://www.getdbt.com/) to convert the document data to a relational database format.
+
+[CHT Pipeline](https://github.com/medic/cht-pipeline) defines a DBT project, which contains model files for the data schema described [in the Database schema conventions]({{< ref "core/overview/db-schema" >}}).
+Forms may be specific to each CHT application; additional models will need to be developed to analyze data from responses to these custom forms.
+One additional model will be needed for each form, and for any aggregations, dashboards, or reusable views that use those form responses as input.
+If using the [configurable contact hierarchy]({{< ref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}), it may also be useful to add models for other contact types.
+
+## Setup
+
+CHT Sync runs models using a [DBT package](https://docs.getdbt.com/docs/build/packages), which is configured by providing a URL to a GitHub repository.
+If application-specific models need to be private, the [cht-pipeline repository](https://github.com/medic/cht-pipeline/) can be forked into a private repository.
+A branch of the public repository can be created if the models can be public.
+
+The URL of the branch is set in the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "apps/guides/data/analytics/environment-variables" >}}), either in ´.env´ if using ´docker-compose´, or in ´values.yaml´ if using Kubernetes.
+
+When models are changed, CHT Sync needs to be restarted for the change to take effect. When it restarts, it automatically downloads the latest version of CHT Pipeline and applies the changes. For models materialized as views, they are updated instantly. For models materialized as incremental tables, the table is dropped and will be unavailable until it is rebuilt from scratch.
+
+## Base Models
+
+The base models define the data which is common to all CHT applications.
+All tables contain a `uuid` which is the primary key for the table; it is also the `_id` from the source CouchDB document.
+
+{{< figure src="cht-pipeline-er.png" link="cht-pipeline-er.png" class=" center col-16 col-lg-12" >}}
```

Comment:
Which tool did you use to generate the diagram? Can you add it to G Drive, share it with the team so anyone can modify it if needed, and add the link to it as a comment in this doc?